CHANGELOG.next.asciidoc

```diff
@@ -170,6 +170,10 @@ field. You can revert this change by configuring tags for the module and omittin
 - system/socket: Fix dataset using 100% CPU and becoming unresponsive in some scenarios. {pull}19033[19033] {pull}19764[19764]
 - system/socket: Fixed tracking of long-running connections. {pull}19033[19033]
 - system/package: Fix librpm loading on Fedora 31/32. {pull}NNNN[NNNN]
+- file_integrity: Create fsnotify watcher only when starting file_integrity module {pull}19505[19505]
```

Comment:
does this one belong here?

Reply:
I'm glad you spotted that. It's fixed now.